datafusion/physical-expr/src/math_expressions.rs

```diff
@@ -701,6 +701,29 @@ macro_rules! make_try_abs_function {
     }};
 }
 
+// Decimal types should keep the same precision and scale by using `with_data_type()`.
+// https://github.com/apache/arrow-rs/issues/4644
```

Comment:
This comment looks useless.

Reply:
Thank you @viirya. Removed since the linked issue has already been closed.